azurelinuxagent/common/utils/cryptutil.py

```diff
@@ -70,6 +72,17 @@ def decrypt_p7m(self, p7m_file, trans_prv_file, trans_cert_file, pem_file):
         rc = shellutil.run(cmd)
         if rc != 0:
             logger.error("Failed to decrypt {0}".format(p7m_file))
+    
+    def decrypt_encrypted_file(self, privateKey, encryptedFile):
+        cmd = "{0} cms -decrypt -inform DER -inkey {1} -in {2}".format(self.openssl_cmd, privateKey, encryptedFile)
+        output = shellutil.run_get_output(cmd)
```

Comment:
rc, output = shellutil.run_get_output(cmd) is better than unpacking an array.

Reply:
Thanks, done.